### PR TITLE
Update various things

### PR DESCRIPTION
**Pull request by Avaq**

### CODE REVIEW

**Comment by Avaq** [comment]
Tests fail in Node 12.

**Comment by dicearr** [comment]
```console
$ nix-shell -p nodejs-12_x --command "npm install && npm run test:unit"

> momi@0.7.2 test:unit /home/diego/projects/warp/momi
> nyc --extension .mjs --reporter=text mocha test/**.test.js


  Momi
    ✓ .run
    ✓ .reject
    ✓ .go
    .fromComputation
      ✓ with a resolved future
      ✓ with a rejected future
    .connect
      ✓ with a resolution in the middleware
      ✓ with a rejection in the middleware
      ✓ with an invalid middleware


  8 passing (16ms)

-----------|----------|----------|----------|----------|-------------------|
File       |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
-----------|----------|----------|----------|----------|-------------------|
All files  |    77.78 |    66.67 |    70.59 |    77.78 |                   |
 index.mjs |    77.78 |    66.67 |    70.59 |    77.78 |... 77,78,79,80,81 |
-----------|----------|----------|----------|----------|-------------------|
```

File: LICENSE [hunk 1]
```diff
@@ -12,8 +12,8 @@ The above copyright notice and this permission notice shall be included in all
...
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
-FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
-COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
-IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
-CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
```
Comment (davidchambers) [inline LICENSE:19]: Why this change?

Comment (Avaq) [reply LICENSE:19]: I used to create files with a character limit set after the 80th character. Ever since I've been using Sanctuary tooling, I've modified this limit to 79 in my editor, to reflect the linting rules. Every once in a while I come across one of those old files, and my editor breaks the lines in them in awkward ways. So I fix it while I'm changing something else in the file.

**Review by dicearr — APPROVED** [review]

File: LICENSE (hunk 1 above)
Comment (davidchambers) [reply LICENSE:19]:
```console
$ grep IMPLIED LICENSE | wc -c
     149
```

Comment (Avaq) [reply LICENSE:19]: Oh. Indeed I have - I missed a single newline. >.<

Comment (Avaq) [reply LICENSE:19]: https://github.com/fluture-js/momi/commit/7f1ac58e9173cf4c9827f959ed66ffa7b26aa637 Thank you for noticing.
